modules/ocf/manifests/packages/vscode.pp

```diff
@@ -0,0 +1,7 @@
+class ocf::packages::vscode {
+  class { 'ocf::packages::vscode::apt':
```

Comment:
perhaps this is a good opportunity to refactor the way we include third-party repositories in our code, instead of duplicating this pattern multiple times for every type of package we want to include from an upstream source instead of the debian repositories.

maybe we can make a single `ocf::packages::third_party` with all the `apt::source` declarations and declared as `stage => first`, and then just add the `code` (and others) packages to `ocf_desktop::packages`.

Reply:
rip vscode is really the time refactor :((

Reply:
Agree. The one drawback is that there would be increased potential for forgetting to remove unneeded third party repositories.

Reply:
Agree, but this is fodder for a separate PR.